stats.php

```diff
@@ -27,20 +34,101 @@
 }
 
 /**
- * Count publish events regardless of post type
+ * Record the created time of a post
  */
-function track_publish_post( $new_status, $old_status ) {
+function record_created_time( $post_id, $post ) {
 	if ( defined( 'WP_IMPORTING' ) && true === WP_IMPORTING ) {
 		return;
 	}
+	// Skip if this is an update, autosave, or revision
+	if ( wp_is_post_autosave( $post_id ) || wp_is_post_revision( $post_id ) ) {
+		return;
+	}
+
+	// Skip if post is auto-draft (but allow draft and other statuses)
+	if ( 'auto-draft' === $post->post_status ) {
+		return;
+	}
+	
+	// Add created time (only if not already set due to $unique = true)
+	add_post_meta( $post_id, '_vip_edtel_created_time', gmdate( 'c' ), true );
+}
 
+function record_published_time( $new_status, $old_status, $post ) {
+	if ( defined( 'WP_IMPORTING' ) && true === WP_IMPORTING ) {
+		return;
+	}
+
+	// Only record when transitioning to publish from non-publish
 	if ( 'publish' !== $new_status || 'publish' === $old_status ) {
 		return;
 	}
 
+	// Skip revisions
+	if ( wp_is_post_revision( $post ) ) {
+		return;
+	}
+
+	// Add published time (only if not already set due to $unique = true)
+	add_post_meta( $post->ID, '_vip_edtel_published_time', gmdate( 'c' ), true );
```

Comment:
Will anyone object to this post meta being set? Are there other VIP-specific post meta?

Reply:
Yeah, we have some sprinkled around in MU....as for objections, probably not since it's not like we are storing a large serialized value?